compiler/rustc_mir_transform/src/simplify.rs

```diff
@@ -290,6 +290,8 @@ pub fn combine_duplicate_switch_targets(terminator: &mut Terminator<'_>) {
     }
 }
 
+// Note: Trying to Turn `duplicates` into a `SmallVec` for performance did not bear fruit, see
+// https://github.com/rust-lang/rust/pull/110524
```

Comment:
Even if we do want to leave a note here, it should probably left as a comment on the field instead?

Reply:
I agree.